docs/generateFeatures.md

```diff
@@ -0,0 +1,55 @@
+## generateFeatures task
+---
+Scan the class files of your application and create a new Liberty configuration file containing the features your application requires.
+
+This task is available as a tech preview in the 3.3.1-SNAPSHOT. Please provide feedback by opening an issue at https://github.com/OpenLiberty/ci.gradle.
+
+This feature is best accessed through dev mode. When you start up `libertyDev` it will compile your application and scan the files to verify that all the required Liberty features are part of your configuration. Then as you work, dev mode will continue to monitor the project to confirm the Liberty features are up to date. If you implement a new interface in Java, the scanner will determine if that API is connected to a Liberty feature, then update the configuration and install the feature. If you remove a feature from `server.xml`, dev mode will determine if that feature is actually necessary, and if so, add it to the configuration file described below. For this snapshot you need to add the Sonatype repository to `build.gradle` (shown below), but in the future all the dependencies will be in Maven Central.
```

Comment:
```suggestion
This feature is best accessed through dev mode. When you start up `libertyDev` it will compile your application and scan the files to verify that all the required Liberty features are part of your configuration. Then as you work, dev mode will continue to monitor the project to confirm the Liberty features are up to date. If you implement a new interface in Java, the scanner will determine if that API is connected to a Liberty feature, then update the configuration and install the feature. If you remove a feature from `server.xml`, dev mode will determine if that feature is actually necessary, and if so, add it to the configuration file as described below. For this snapshot you need to add the Sonatype repository to `build.gradle` (shown below), but in the future all the dependencies will be in Maven Central.
```